### PR TITLE
fix AMP stats panic

### LINKED ISSUE #531: amp stats panic
Sometimes, the command `amp stats` causes a panic and crashes amplifier.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xc0 pc=0x544af5]

goroutine 583 [running]:
panic(0xc1c960, 0xc42000c060)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/appcelerator/amp/api/rpc/stats.StatsReply.Less(0xc42027e280, 0xf, 0xf, 0xe, 0x0, 0x0)
        /go/src/github.com/appcelerator/amp/api/rpc/stats/stats.go:452 +0x45
github.com/appcelerator/amp/api/rpc/stats.(*StatsReply).Less(0xc420696c20, 0xe, 0x0, 0x0)
        <autogenerated>:3 +0x78
sort.medianOfThree(0x13de0c0, 0xc420696c20, 0x0, 0x7, 0xe)
        /usr/local/go/src/sort/sort.go:78 +0x7e
sort.doPivot(0x13de0c0, 0xc420696c20, 0x0, 0xf, 0x19, 0x0)
        /usr/local/go/src/sort/sort.go:103 +0xa7
sort.quickSort(0x13de0c0, 0xc420696c20, 0x0, 0xf, 0x7)
        /usr/local/go/src/sort/sort.go:188 +0x83
sort.Sort(0x13de0c0, 0xc420696c20)
        /usr/local/go/src/sort/sort.go:222 +0x80
github.com/appcelerator/amp/api/rpc/stats.(*Stats).StatsQuery(0xc4203681e0, 0x13e2fc0, 0xc4207233b0, 0xc42098aa20, 0x0, 0xb, 0xb)
        /go/src/github.com/appcelerator/amp/api/rpc/stats/stats.go:68 +0x1a2
github.com/appcelerator/amp/api/rpc/stats._Stats_StatsQuery_Handler(0xce9d80, 0xc4203681e0, 0x13e2fc0, 0xc4207233b0, 0xc4205631d0, 0x0, 0x0, 0x0, 0xc42006e358, 0x0)
        /go/src/github.com/appcelerator/amp/api/rpc/stats/stats.pb.go:470 +0x27d
github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).processUnaryRPC(0xc420520780, 0x13e5b60, 0xc4208605a0, 0xc420218ff0, 0xc42050cfc0, 0x13ba710, 0xc420723380, 0x0, 0x0)
        /go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:638 +0xc14
github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).handleStream(0xc420520780, 0x13e5b60, 0xc4208605a0, 0xc420218ff0, 0xc420723380)
        /go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:796 +0x6b0
github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc420a8c7b0, 0xc420520780, 0x13e5b60, 0xc4208605a0, 0xc420218ff0)
        /go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:449 +0xab
created by github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
        /go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:450 +0xa3
```

### PR DESCRIPTION
**Pull request by freignat91**
Fix #531 

test:
- ./shrink.sh local
- make install
- amp pf start --local -f
- make test

### CODE REVIEW

**Review by bquenin — APPROVED** [review]
Took a while, but I think it's good now

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # appcelerator/protoc is based on alpine and includes latest go and protoc
```
Comment (ndegory) [inline Dockerfile]: we should remove that comment, it doesn't apply anymore

**Review by ndegory — APPROVED** [review]
LGTM, tests passing on Mac OS

File: Dockerfile (hunk 1 above)
Comment (bquenin) [reply Dockerfile]: Ok

**Review by JosephGJ — APPROVED** [review]
LGTM

**Pull request merged**